swarm/src/behaviour/peer_addresses.rs

```diff
@@ -8,16 +8,39 @@ use lru::LruCache;
 
 use std::num::NonZeroUsize;
 
+#[derive(Debug, Clone)]
+/// Configuration of a [`PeerAddresses`] instance.
+pub struct PeerAddressesConfig {
+    /// Capacity of the [`PeerAddresses`] cache.
+    pub number_of_peers: NonZeroUsize,
+
+    /// Maximum number of cached addresses per peer.
+    pub number_of_addresses_by_peer: NonZeroUsize,
+}
+
+impl Default for PeerAddressesConfig {
+    fn default() -> Self {
+        Self {
+            number_of_peers: NonZeroUsize::new(100).expect("100 != 0"),
```

Comment:
Is this number based on identify cache size or what there another thought process to determine this default value?

Reply:
It is based both on the default identify cache size and the default `number_of_peers` value of `PeerAddresses` which are both `100`.